### PR TITLE
Document how to configure PSP

### PR DESCRIPTION
**Pull request by aojea**

### CODE REVIEW

File: site/content/docs/user/pod-security-policy.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+---
+
+# Pod Security Policy
+
+This guide covers setting up [Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/)
+on a kind cluster.
+
+## Enabling Pod Security Policy
+
+Pod security policy control is implemented as an optional (but recommended) admission controller. PodSecurityPolicies are enforced by enabling the admission controller, but doing so without authorizing any policies will prevent any pods from being created in the cluster.
```
Comment (neolit123) [inline site/content/docs/user/pod-security-policy.md]: i'd add a note here that PSP *might be going away*. not sure where the slides are, but you can link to the sig-auth agenda document - e.g. "SIG Auth is having discussions about ...."

File: site/content/docs/user/pod-security-policy.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+However, the cluster is not ready to be used at this stage, the nodes are not ready:
+
+{{< codeFromInline lang="bash" >}}
+kubectl get nodes -A
+NAME                 STATUS     ROLES    AGE   VERSION
+kind-control-plane   NotReady   master   94s   v1.17.2
+kind-worker          NotReady   <none>   52s   v1.17.2
+{{< /codeFromInline >}}
+
+and there is no Pods:
```
Comment (neolit123) [inline site/content/docs/user/pod-security-policy.md]: there are

File: site/content/docs/user/pod-security-policy.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+{{< codeFromInline lang="bash" >}}
+kubectl get pods -A
+No resources found.
+{{< /codeFromInline >}}
+
+We need to grant the controller access to the policy so it can create the system pods, like CoreDNS, kube-scheduler, ...
```
Comment (neolit123) [inline site/content/docs/user/pod-security-policy.md]: `, ...` -> `, etc.`

File: site/content/docs/user/pod-security-policy.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+{{< /codeFromInline >}}
+
+
+after a few seconds you can observe how the system pods are being created and your cluster is ready to be used with Pod Security Policy enabled.
+
+{{< codeFromInline lang="bash" >}}
+kubectl get pods -A
+NAMESPACE            NAME                                         READY   STATUS    RESTARTS   AGE
+kube-system          coredns-6955765f44-f5572                     0/1     Pending   0          25s
+kube-system          coredns-6955765f44-tt2fl                     0/1     Pending   0          25s
```
Comment (neolit123) [inline site/content/docs/user/pod-security-policy.md]: potentially the example can show all the pods as 1/1 Ready, Running.

File: site/content/docs/user/pod-security-policy.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+No resources found.
+{{< /codeFromInline >}}
+
+We need to grant the controller access to the policy so it can create the system pods, like CoreDNS, kube-scheduler, ...
+
+Deploy [Authorization Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#authorizing-policies).
+
+{{< codeFromInline lang="bash" >}}
+cat <<EOF | kubectl apply -f -
+piVersion: policy/v1beta1
```
Comment (neolit123) [inline site/content/docs/user/pod-security-policy.md]: apiVersion

File: site/content/docs/user/pod-security-policy.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+and there is no Pods:
+
+{{< codeFromInline lang="bash" >}}
+kubectl get pods -A
+No resources found.
+{{< /codeFromInline >}}
+
+We need to grant the controller access to the policy so it can create the system pods, like CoreDNS, kube-scheduler, ...
+
+Deploy [Authorization Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#authorizing-policies).
```
Comment (neolit123) [inline site/content/docs/user/pod-security-policy.md]: might be a good idea to indicate that this example policy is a non-restrictive one.

File: site/content/docs/user/pod-security-policy.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
+---
+title: "Pod Security Policy"
+menu:
+  main:
+    parent: "user"
+    identifier: "pod-security-policy"
+    weight: 3
+---
+
+# Pod Security Policy
```
Comment (BenTheElder) [inline site/content/docs/user/pod-security-policy.md]: This needs to be clearly unsupported if we're going to document this.
I do not want to spend cycles on issues with broken PSP setups, especially given that it is not getting support from kubeadm.

**Comment by aojea** [comment]
/close
you have a point, despite we can mention that is unsupported once it's documented questions are going to come.
Due to the uncertainty of this feature, I think that is better to close it.
If somebody wants to try they can always refer to this closed PR :man_shrugging:

**Pull request closed without merging**